pkg/prowgen/prowgen_config.go

```diff
@@ -14,28 +14,51 @@ import (
 type Repository struct {
 	Org                   string                                                      `json:"org" yaml:"org"`
 	Repo                  string                                                      `json:"repo" yaml:"repo"`
+	Promotion             Promotion                                                   `json:"promotion" yaml:"promotion"`
 	ImagePrefix           string                                                      `json:"imagePrefix" yaml:"imagePrefix"`
 	ImageNameOverrides    map[string]string                                           `json:"imageNameOverrides" yaml:"imageNameOverrides"`
 	SlackChannel          string                                                      `json:"slackChannel" yaml:"slackChannel"`
 	CanonicalGoRepository *string                                                     `json:"canonicalGoRepository" yaml:"canonicalGoRepository"`
-	E2ETests              E2ETests                                                    `json:"e2e" yaml:"e2e"`
+	E2ETests              []E2ETest                                                   `json:"e2e" yaml:"e2e"`
```

Comment:
@mgencur why you had to change this one?

Reply:
@skonto Because now it's possible to set various things for individual tests as can be seen here: https://github.com/openshift-knative/hack/pull/84/files#diff-9cb0409bd8a2445656e16ee55c0cc76205f9e9753d4cafa7d59d9bd879395ffdR30
"Match" means the regexp pattern.
Example usage: https://github.com/openshift-knative/hack/pull/84/files#diff-8c104313544b2c9a8413d2cbb7a399e56793ffab4a3e554d5ac2243a5a74f1c8R53